packages/website/components/accordionblock/accordionblock.js

```diff
@@ -18,7 +20,11 @@ export default function AccordionBlock({ block }) {
               <div className="accordion-header-text">{section.heading}</div>
             </ZeroAccordionSection.Header>
             <ZeroAccordionSection.Content>
-              <div className="accordion-content-text" dangerouslySetInnerHTML={{ __html: section.content }}></div>
+              {block.markdown?.includes('content') ? (
+                <ReactMarkdown>{section.content}</ReactMarkdown>
+              ) : (
+                <div className="accordion-content-text" dangerouslySetInnerHTML={{ __html: section.content }}></div>
```

Comment:
I know this was already there, so I'm not saying we have to change it now, but this really is quite dangerous. If possible over time it seems like we should be able to avoid it, e.g. by passing `props.children` as a `React.Element` for this content instead of raw html strings.

Reply:
I could import a new package (for example DOMPurify) and pass it through there first.  Would that work?

Reply:
I don't think that's worth adding dep on dompurify vs avoiding authoring/passing-around html strings in the first place. JSX helps with sanitization, we just have to use it.